internal/metrics.go

```diff
@@ -0,0 +1,121 @@
+package internal
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/cactus/go-statsd-client/v5/statsd"
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+	"github.com/spf13/viper"
+	"github.com/wal-g/tracelog"
+)
+
+var (
```

Comment:
Let's introduce some common struct, e.g. `metrics` to hold all the current and future metrics?

Reply:
This is the only one I wasn't sure about. If we want to introduce more low-level metrics for a particular database, we may want to create a separate `metrics.go` inside that database package.

Of course, if you think we'd want a metrics struct per database in this case, I'm happy to introduce the struct.